src/config/generic_development_dependency.go

```diff
@@ -38,5 +39,10 @@ func (g *genericDevelopmentDependency) GetDockerConfig() (types.DockerConfig, er
 // GetServiceEnvVariables returns the environment variables that need to
 // be passed to services that use it
 func (g *genericDevelopmentDependency) GetServiceEnvVariables() map[string]string {
-	return g.config.Config.ServiceEnvironment
+	result := map[string]string{}
+	result[strings.ToUpper(g.config.Name)] = g.GetContainerName()
+	for key, value := range g.config.Config.ServiceEnvironment {
+		result[key] = value
+	}
+	return result
```

Comment:
Can this be reversed to:
```
result := g.config.Config.ServiceEnvironment
result[strings.ToUpper(g.config.Name)] = g.GetContainerName()
return result
```
?

Reply:
That would edit the ServiceEnviorment object which i would prefer not to do. Believe I can use `util.Merge` though.